src/api/session/base/Scheduler.cpp

```diff
@@ -73,6 +76,7 @@ void Scheduler::Start(
         [=, &dht]() -> void {
             ServerLambda serverLambda(
                 [=, &dht](const proto::ServerContract& server) -> void {
+                    SetThisThreadsName("Scheduler publish server");
```

Comment:
Very long, embedded blanks, these names are no good.
Perhaps they should be predefined somewhere in a separate file like
constexpr std::string_view thrName_Whatever {"Whatever\0"sv};
static_assert(thrName_Whatever.size < 16);
and here the verified symbol should be used, i.e. thrName_Whatever, for example?